shiny/express/_run.py

```diff
@@ -115,7 +115,7 @@ def set_result(x: object):
         if "app" in var_context and isinstance(var_context["app"], App):
             raise RuntimeError(
                 "This looks like a Shiny Express app because it imports shiny.express, "
-                "but it also looks like a Shiny Classic app because it has a variable named "
+                "but it also looks like a Shiny Core app because it has a variable named "
```

Comment:
This is kind of unrelated to the create process, but I was greping for occurrences of Classic and thought I'd change it.